packages/documentation/stories/components/alert/alert.stories.tsx

```diff
@@ -0,0 +1,238 @@
+import React from 'react';
+import parse from 'html-react-parser';
+import { useArgs } from '@storybook/client-api';
+import docsPage from './alert.docs.mdx';
+
+import '@swisspost/design-system-styles/basics.scss';
+import '@swisspost/design-system-styles/components/alert.scss';
+import '@swisspost/design-system-styles/components/icons.scss';
+import './alert.stories.scss';
+
+export default {
+  title: 'Components/Alert',
+  parameters: {
+    docs: {
+      page: docsPage
+    },
+    controls: {
+      exclude: [
+        'Icon',
+        'Icon Color',
+        'No Icon',
+        'Dismissible',
+        'Fixed'
+      ]
+    }
+  },
+  args: {
+    title: 'Titulum',
+    content: '<p>Contentus momentus vero siteos et accusam iretea et justo.</p>',
+    variant: 'alert-primary',
+    icon: '',
+    iconVariant: '',
+    noIcon: false,
+    dismissible: false,
+    dismiss: false,
+    fixed: false,
+    show: false
+  },
+  argTypes: {
+    title: {
+      name: 'Title',
+      control: { type: 'text' },
+      table: { category: 'Content' }
+    },
+    content: {
+      name: 'Content',
+      control: { type: 'text' },
+      table: { category: 'Content' }
+    },
+    variant: {
+      name: 'Variant',
+      control: {
+        type: 'select',
+        labels: {
+          'alert-primary': 'Primary',
+          'alert-notification': 'Notification',
+          'alert-success': 'Success',
+          'alert-danger': 'Danger',
+          'alert-error': 'Error',
+          'alert-warning': 'Warning',
+          'alert-info': 'Info'
+        }
+      },
+      options: [
+        'alert-primary',
+        'alert-notification',
+        'alert-success',
+        'alert-danger',
+        'alert-error',
+        'alert-warning',
+        'alert-info'
+      ],
+      table: { category: 'Variant' }
+    },
+    icon: {
+      name: 'Icon',
+      control: {
+        type: 'select',
+        labels: {
+          '': 'Default',
+          '1001': 'Envelope',
+          '2023': 'Cog',
+          '2025': 'Send',
+          '2035': 'Home',
+          '2101': 'Bubble'
+        }
+      },
+      options: [
+        '',
+        '1001',
+        '2023',
+        '2025',
+        '2035',
+        '2101'
+      ],
+      table: { category: 'Icon' }
+    },
+    iconVariant: {
+      name: 'Icon Color',
+      control: {
+        type: 'select',
+        labels: {
```

Comment:
Icon color on the alert should be one of the contrast colors, devs should not color it differently in order to keep the color contrast as high as possible. There's functionality in place to automatically handle this in high contrast mode as well using `filter: invert(1)`. I'd not provide the option to color icons here.

Reply:
This is control is only there if you integrate a custom icon (which is by default dark). Don't you think it is important to show the possibility to change the icon color in this case?

Reply:
This would be part of the icon documentation